lib/avo/configuration.rb

```diff
@@ -231,6 +235,21 @@ def default_logger
       }
     end
 
+    def database_exists?
+      ActiveRecord::Base.connection
+    rescue ActiveRecord::NoDatabaseError
+      false
+    else
+      true
+    end
+
+    def audit?
```

Comment:
I don't see where we use this method, probably in the gem itself?
If so, how about moving it there so there's little noise about it here?

Reply:
Yes, I'm moving the audit configuration to the gem and will revert this change, thanks for pointing it